cosmic-settings/src/pages/input/keyboard/shortcuts/mod.rs

```diff
@@ -732,6 +732,7 @@ fn localize_action(action: &Action) -> String {
             SystemAction::PlayPrev => fl!("system-shortcut", "play-prev"),
             SystemAction::PowerOff => fl!("system-shortcut", "poweroff"),
             SystemAction::Screenshot => fl!("system-shortcut", "screenshot"),
+            SystemAction::Suspend => fl!("system-shortcut", "suspend"),
```

Comment:
It looks like this is supposed to show up as a shortcut on the System page of the keyboard shortcut settings? I'm not seeing it in there. Is it supposed to be visible, or is this just used for something else?

Reply:
It makes it possible to localize, but indeed it needs to be manually added to that category to appear there.

Reply:
It makes it possible to localize, but indeed it needs to be manually added to that category to appear there.

Reply:
93164e6 makes it show up in search without having to be set first, like all other default shortcuts.